### DISCUSSION TITLE
MIDI keyboard not recognized

### DISCUSSION THREAD

**Discussion by lucpotage** [GrandOrgue/grandorgue#1285] [Help] [answered]
I tried to use a MIDI keyboard with GrandOrgue but it did not work. I tried another synth (Mela 2) and it worked.

Here is my setup:
- macOS Ventura 13.0
- GrandOrgue 3.9.1-1

<img width="911" alt="Capture d’écran 2022-11-27 à 17 20 38" src="https://user-images.githubusercontent.com/7230764/204145982-b02ba180-eabf-49d9-ace1-49fc083ebc0d.png">

**Comment by oleg68** [comment, marked as answer]
You have assign the keyboard to a manual. Right-ckick on the manual, click on `Listen to Event` and press any key on the keyboard.

**Comment by lucpotage** [comment]
Thank you it worked!
